src/Minsk/CodeAnalysis/Syntax/CompoundAssignmentExpressionSyntax.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Minsk.CodeAnalysis.Syntax
+{
+    public sealed partial class CompoundAssignmentExpressionSyntax : ExpressionSyntax
```

Comment:
I think we should reuse the `AssingmentExpressionSyntax` because the shape will be identical.

Reply:
Done I also renamed `EqualsToken` property in `AssingmentExpressionSyntax` to `AssignmentToken`